.github/workflows/run-test-cases.yml

```diff
@@ -117,6 +117,8 @@ jobs:
             crictl: v1.17.0
           - runtime: MicroK8s-1.20
             version: 1.20/stable
```

Comment:
looks like microk8s-1.20 lost its `crictl: v1.17.0`

Reply:
Thank you!